admin-console/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -33,10 +29,10 @@ export const ApplicationPage: React.FunctionComponent = () => {
     const eventTypesQuery = useEventTypes(applicationId);
     const applicationTypesQuery = useApplicationTypes(applicationId);
     const deleteEventTypeMutation = useDeleteEventType();
+    const newEvent = useCreateEventType();
 
     const columns = [ 'Event Type', 'Name', 'Description', 'Event Type Id' ];
 
-    const newEvent = useCreateEventType();
     const [ eventType, setEventType ] = React.useState<Partial<EventType>>({});
```

Comment:
I would preffer to have this in the modal as well.

Reply:
this currently is not being used in the modal, but instead here in the handleSubmit...

Reply:
Yes, that's what i mean. instead of using this variable here, just pass it to the `onSubmit`  (from the modal)

Reply:
I moved this to the modal, but trying to understand. You're saying pass it to the new onsubmitLocal from the modal?

Reply:
The new `onSubmitLocal` should pass it to the `onSubmit` that is in `ApplicationPage` so it will be something like:
In ApplicationPage:

// ```
onSubmit(eventType: EventType) {
  // ...
}
// ....
<CreateEditModal onSubmit={ onSubmit }  />
```

and in CreateEditModal
```
// ...
onSubmitLocal = () => {
  // ...
  props.onSubmit(eventType);
}
```